src/page/blog/draft-page.jsx

```diff
@@ -30,12 +30,16 @@ export default function Drafts() {
 
   return (
     <>
-      <div className="w-full max-auto max-w-2xl h-40 left-60 relative">
+      <div
+        className="w-full max-auto max-w-2xl h-40 left-60 relative"
+        data-test-id="articles-page-draft"
```

Comment:
```suggestion
        data-test-id="articles-page-drafts"
```
Since this is a list of drafts inside of this draft, it would be better to make it plural.